RNApp/app/routes/MapView/MapView.js

```diff
@@ -3,6 +3,7 @@ import { Text, View } from 'react-native';
 import Button from '../../components/Button';
 import styles from './styles';
 import Map from './../../components/Map';
+// import { makeRequest, getCountryName } from './../../backend/tempBackend';

```

Comment:
Also -- why is this line commented out? Is it needed to preview the changes? Does this line need to be included in our `master` branch?